### PR TITLE
Simplify multi project instance by removing many makefile creation

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: doc/integrator/create_application.rst [hunk 1]
```diff
@@ -489,11 +482,20 @@ Create the project
    +dest = /var/www/vhosts/<project>/conf/${instance}.conf
    +content = Include /var/www/vhosts/<project>/private/${instance}/apache/*.conf
 
-12. In ``.gitignore`` add the following lines:
+12. Update the deploy configuration as follow:
```
Comment (asaunier) [inline doc/integrator/create_application.rst]: as follows

**Comment by asaunier** [comment]
Just curious why the _jinja extension are no longer needed?

**Comment by sbrunner** [comment]
Because we no longer create a template of makefile that create a set of make file, we change it to have a generic makefile and select the right project with an environment variable.

**Comment by asaunier** [comment]
OK thanks for the explanation.
LGTM

**Pull request merged**